modules/lightmapper_rd/lm_compute.glsl

```diff
@@ -249,6 +249,15 @@ float quick_hash(vec2 pos) {
 	return fract(sin(dot(pos * 19.19, vec2(49.5791, 97.413))) * 49831.189237);
 }
 
+float get_omni_attenuation(float distance, float inv_range, float decay) {
+	float nd = distance * inv_range;
+	nd *= nd;
+	nd *= nd; // nd^4
```

Comment:
Isn't this `nd^3`?

Reply:
I thought this initially too but no, it's equivalent to:
```
float nd2 = nd * nd;
float nd4 = nd2 * nd2;
```